### ISSUE TITLE
0.31.8. Multi nvidia GPU error on start - PLSR 

### ISSUE THREAD

**Issue by deocP** [andru-kun/wildrig-multi#139]
```
 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[09:33:41] initializing GPUs...
[09:33:41] PTX ISA: 7.6
[09:33:41] GPU #0: NVIDIA GeForce GTX 1660 SUPER [busID: 2] [arch: sm75]
[09:33:41] threads: 1, intensity: 18, worksize: 64/1024, cu: 44, mem: 6143Mb
[09:33:43] GPU #1: NVIDIA GeForce RTX 2060 [busID: 3] [arch: sm75]
[09:33:43] threads: 1, intensity: 18, worksize: 64/1024, cu: 60, mem: 6143Mb
[09:33:43] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[09:33:43] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:33:45] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[09:33:45] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:33:45] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[09:33:45] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:33:46] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[09:33:46] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:33:46] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[09:33:46] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[09:33:46] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[09:33:46] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:33:47] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[09:33:47] Stratum set raw difficulty to 0.0001
[09:33:47] new job from stratum-eu.rplant.xyz:7030 diff 430k/29.38G
[09:33:47] block: 341,570       job target: 0x0000270fd8f00000
[09:33:47] CPU: precompute calculation
[09:33:51] GPU #1: start fill precompute
[09:33:51] GPU #4: start fill precompute
[09:33:51] GPU #5: start fill precompute
[09:33:51] GPU #0: start fill precompute
[09:33:51] GPU #3: start fill precompute
[09:33:51] GPU #2: start fill precompute
[09:33:51] GPU #7: start fill precompute
[09:33:51] GPU #6: start fill precompute
[09:33:52] GPU #0: precompute finished

Waiting for 0 seconds, press a key to continue ...

 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[09:33:58] initializing GPUs...
[09:33:59] PTX ISA: 7.6
[09:33:59] GPU #0: NVIDIA GeForce GTX 1660 SUPER [busID: 2] [arch: sm75]
[09:33:59] threads: 1, intensity: 18, worksize: 64/1024, cu: 44, mem: 6143Mb
[09:33:59] GPU #1: NVIDIA GeForce RTX 2060 [busID: 3] [arch: sm75]
[09:33:59] threads: 1, intensity: 18, worksize: 64/1024, cu: 60, mem: 6143Mb
[09:33:59] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[09:33:59] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:00] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[09:34:00] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:34:00] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[09:34:00] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:34:00] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[09:34:00] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:01] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[09:34:01] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[09:34:01] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[09:34:01] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:02] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[09:34:02] Stratum set raw difficulty to 0.0001
[09:34:02] new job from stratum-eu.rplant.xyz:7030 diff 430k/29.38G
[09:34:02] block: 341,570       job target: 0x0000270fd8f00000
[09:34:02] CPU: precompute calculation
[09:34:04] new job from stratum-eu.rplant.xyz:7030 diff 430k/29.38G
[09:34:04] block: 341,570       job target: 0x0000270fd8f00000
[09:34:06] GPU #0: start fill precompute
[09:34:06] GPU #3: start fill precompute
[09:34:06] GPU #6: start fill precompute
[09:34:06] GPU #4: start fill precompute
[09:34:06] GPU #1: start fill precompute
[09:34:06] GPU #5: start fill precompute
[09:34:06] GPU #2: start fill precompute
[09:34:06] GPU #7: start fill precompute
[09:34:07] GPU #7: precompute finished
[09:34:07] GPU #5: precompute finished

Waiting for 0 seconds, press a key to continue ...

 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[09:34:13] initializing GPUs...
[09:34:14] PTX ISA: 7.6
[09:34:14] GPU #0: NVIDIA GeForce GTX 1660 SUPER [busID: 2] [arch: sm75]
[09:34:14] threads: 1, intensity: 18, worksize: 64/1024, cu: 44, mem: 6143Mb
[09:34:14] GPU #1: NVIDIA GeForce RTX 2060 [busID: 3] [arch: sm75]
[09:34:14] threads: 1, intensity: 18, worksize: 64/1024, cu: 60, mem: 6143Mb
[09:34:14] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[09:34:14] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:15] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[09:34:15] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:34:15] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[09:34:15] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[09:34:15] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[09:34:15] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:16] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[09:34:16] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[09:34:16] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[09:34:16] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[09:34:17] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[09:34:17] Stratum set raw difficulty to 0.0001
[09:34:17] new job from stratum-eu.rplant.xyz:7030 diff 430k/29.38G
[09:34:17] block: 341,570       job target: 0x0000270fd8f00000
[09:34:17] CPU: precompute calculation
[09:34:21] GPU #0: start fill precompute
[09:34:21] GPU #3: start fill precompute
[09:34:21] GPU #2: start fill precompute
[09:34:21] GPU #4: start fill precompute
[09:34:21] GPU #5: start fill precompute
[09:34:21] GPU #1: start fill precompute
[09:34:21] GPU #7: start fill precompute
[09:34:21] GPU #6: start fill precompute
[09:34:22] GPU #1: precompute finished
[09:34:22] GPU #7: precompute finished

Waiting for 2
```

**Comment by andru-kun** [comment]
try to use parameter --scratchpad-safe-update

**Comment by andru-kun** [comment]
try to run miner on RTX 3xxx series separately, with parameter -d 2,3,4,5,6,7

**Comment by deocP** [comment]
First run it crashed, at CPU precompute....
Sedond hangs at 


 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[10:49:39] initializing GPUs...
[10:49:39] PTX ISA: 7.6
[10:49:39] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[10:49:39] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:49:40] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[10:49:40] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:49:40] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[10:49:40] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:49:41] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[10:49:41] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:49:41] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[10:49:41] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[10:49:41] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[10:49:41] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:49:42] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:49:42] Stratum set raw difficulty to 0.0001
[10:49:42] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.80G
[10:49:42] block: 341,687       job target: 0x0000270fd8f00000
[10:49:42] CPU: precompute calculation
[10:49:56] new job from stratum-eu.rplant.xyz:7030 diff 430k/29.40G
[10:49:56] block: 341,688       job target: 0x0000270fd8f00000
[10:49:56] GPU #6: start fill precompute
[10:49:56] GPU #4: start fill precompute
[10:49:56] GPU #2: start fill precompute
[10:49:56] GPU #5: start fill precompute
[10:49:56] GPU #7: start fill precompute
[10:49:56] GPU #3: start fill precompute
[10:49:57] GPU #7: precompute finished
[10:49:57] GPU #2: precompute finished
[10:49:57] GPU #5: precompute finished
[10:49:58] GPU #7: yay! found 6 shares at once
[10:49:58] rejected(0/1)        diff 830k       GPU#7   (60 ms)
[10:49:58] reason: bad share
[10:49:58] GPU #5: yay! found 3 shares at once
[10:49:58] GPU #2: yay! found 6 shares at once
[10:49:58] rejected(0/2)        diff 779k       GPU#7   (206 ms)
[10:49:58] reason: bad share
[10:49:58] rejected(0/3)        diff 554k       GPU#7   (207 ms)
[10:49:58] reason: bad share
[10:49:58] rejected(0/4)        diff 830k       GPU#7   (208 ms)
[10:49:58] reason: bad share
[10:49:58] oh my gosh! Too many rejects, disconnect
[10:49:59] rejected(0/5)        diff 505k       GPU#7   (209 ms)
[10:49:59] reason: bad share
[10:49:59] GPU #7: yay! found 5 shares at once
[10:49:59] GPU #5: yay! found 6 shares at once
[10:49:59] GPU #2: yay! found 6 shares at once
[10:50:01] GPU #3: precompute finished
[10:50:01] GPU #6: precompute finished
[10:50:01] GPU #4: precompute finished
[10:50:39] Uptime: 0 days 00:00:59
[10:50:39] GPU #2[T:37C P: 30W F:34%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #3[T:53C P: 26W F: 0%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #4[T:39C P: 29W F:32%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #5[T:34C P: 30W F:33%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #6[T:53C P: 83W F:38%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #7[T:43C P: 38W F:35%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] hashrate: 10s: n/a 60s: n/a 15m: n/a kH/s max: 884.27 kH/s

**Comment by deocP** [comment]
........

[10:49:58] oh my gosh! Too many rejects, disconnect
[10:49:59] rejected(0/5)        diff 505k       GPU#7   (209 ms)
[10:49:59] reason: bad share
[10:49:59] GPU #7: yay! found 5 shares at once
[10:49:59] GPU #5: yay! found 6 shares at once
[10:49:59] GPU #2: yay! found 6 shares at once
[10:50:01] GPU #3: precompute finished
[10:50:01] GPU #6: precompute finished
[10:50:01] GPU #4: precompute finished
[10:50:39] Uptime: 0 days 00:00:59
[10:50:39] GPU #2[T:37C P: 30W F:34%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #3[T:53C P: 26W F: 0%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #4[T:39C P: 29W F:32%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #5[T:34C P: 30W F:33%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #6[T:53C P: 83W F:38%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] GPU #7[T:43C P: 38W F:35%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:50:39] hashrate: 10s: n/a 60s: n/a 15m: n/a kH/s max: 884.27 kH/s
[10:51:39] Uptime: 0 days 00:01:59
[10:51:39] GPU #2[T:37C P: 29W F:34%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] GPU #3[T:56C P: 27W F: 0%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] GPU #4[T:39C P: 29W F:32%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] GPU #5[T:34C P: 30W F:33%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] GPU #6[T:52C P: 83W F:38%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] GPU #7[T:42C P: 38W F:35%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:39] hashrate: 10s: n/a 60s: n/a 15m: n/a kH/s max: 884.27 kH/s
[10:51:49] Uptime: 0 days 00:02:09
[10:51:49] GPU #2[T:37C P: 29W F:34%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] GPU #3[T:56C P: 27W F: 0%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] GPU #4[T:39C P: 29W F:32%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] GPU #5[T:34C P: 30W F:33%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] GPU #6[T:52C P: 83W F:38%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] GPU #7[T:42C P: 38W F:35%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:51:49] hashrate: 10s: n/a 60s: n/a 15m: n/a kH/s max: 884.27 kH/s

**Comment by deocP** [comment]
OK it's running - I have to RUN it 4th TIMES to work - waiting for diff change from stratum...



 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[10:52:36] initializing GPUs...
[10:52:36] PTX ISA: 7.6
[10:52:36] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[10:52:36] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:52:37] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[10:52:37] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:52:37] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[10:52:37] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:52:38] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[10:52:38] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:52:38] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[10:52:38] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[10:52:38] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[10:52:38] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:53:36] Uptime: 0 days 00:00:59
[10:53:36] GPU #2[T:37C P: 29W F:34%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] GPU #3[T:59C P: 29W F: 0%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] GPU #4[T:39C P: 29W F:32%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] GPU #5[T:34C P: 30W F:33%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] GPU #6[T:52C P: 83W F:38%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] GPU #7[T:41C P: 38W F:35%]: 10s: n/a 60s: n/a 15m: n/a kH/s
[10:53:36] hashrate: 10s: n/a 60s: n/a 15m: n/a kH/s max: n/a kH/s
[10:53:45] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:53:45] Stratum set raw difficulty to 0.0001
[10:53:45] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.20G
[10:53:45] block: 341,694       job target: 0x0000270fd8f00000
[10:53:45] CPU: precompute calculation
[10:53:49] GPU #2: start fill precompute
[10:53:49] GPU #7: start fill precompute
[10:53:49] GPU #4: start fill precompute
[10:53:49] GPU #5: start fill precompute
[10:53:49] GPU #6: start fill precompute
[10:53:49] GPU #3: start fill precompute
[10:53:50] GPU #5: precompute finished
[10:53:50] GPU #2: precompute finished
[10:53:50] GPU #7: precompute finished

Waiting for 0 seconds, press a key to continue ...

 ###################################
 #################  ################
 ################    ############### * ABOUT:    WildRig/0.31.8 libuv/1.21.0 OpenCL/2.0 MSVC/2015
 ####   #########    #########   ### * CPU:      Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
 ####     #######    #######     ### * ALGO:     curvehash
 ####   #  ######    ######      ### * POOL #1:  stratum-eu.rplant.xyz:7030
 ####   ##  #####    #####  ##   ### * COMMANDS: hashrate, statistics, pause, resume
 ####   ##   #####  #####   ##   ###
 ####   ##   #####  #####  ###   ###   donation: 1%
 ####   ###  #####  #####  ###   ###   send-stale: no
 ####   ###  ####    ####  ###   ###   watchdog: disabled
 ####   ###   ###    ###   ###   ###   temp limits: 60C..85C
 ####   ##    ##      ##    ##   ###
 ####      #####  ##  #####      ###
 ####   #   ###  ####  ###   #   ###
 ####   ##       ####       ##   ###
 ####   #####  #      #  #####   ###
 ####   #####  ###  ###  #####   ###
 ####                            ###
 ############   ##  ##   ###########
 #############   #  #   ############
  #############        #############
      ###########    ###########
          ##################

[10:53:56] initializing GPUs...
[10:53:56] PTX ISA: 7.6
[10:53:56] GPU #2: NVIDIA GeForce RTX 3060 Ti [busID: 4] [arch: sm86]
[10:53:56] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:53:57] GPU #3: NVIDIA GeForce RTX 3060 [busID: 6] [arch: sm86]
[10:53:57] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:53:57] GPU #4: NVIDIA GeForce RTX 3060 [busID: 7] [arch: sm86]
[10:53:57] threads: 1, intensity: 21, worksize: 64/1024, cu: 56, mem: 12287Mb
[10:53:58] GPU #5: NVIDIA GeForce RTX 3060 Ti [busID: 8] [arch: sm86]
[10:53:58] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:53:58] GPU #6: NVIDIA GeForce RTX 3080 [busID: 9] [arch: sm86]
[10:53:58] threads: 1, intensity: 21, worksize: 64/1024, cu: 136, mem: 10239Mb
[10:53:58] GPU #7: NVIDIA GeForce RTX 3060 Ti [busID: 10] [arch: sm86]
[10:53:58] threads: 1, intensity: 21, worksize: 64/1024, cu: 76, mem: 8191Mb
[10:53:59] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:53:59] Stratum set raw difficulty to 0.0001
[10:53:59] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.20G
[10:53:59] block: 341,695       job target: 0x0000270fd8f00000
[10:53:59] CPU: precompute calculation
[10:54:14] GPU #3: start fill precompute
[10:54:14] GPU #7: start fill precompute
[10:54:14] GPU #6: start fill precompute
[10:54:14] GPU #4: start fill precompute
[10:54:14] GPU #2: start fill precompute
[10:54:14] GPU #5: start fill precompute
[10:54:15] GPU #5: precompute finished
[10:54:15] GPU #7: precompute finished
[10:54:15] GPU #2: precompute finished
[10:54:16] rejected(0/1)        diff 1.86M      GPU#7   (56 ms)
[10:54:16] reason: bad share
[10:54:16] GPU #5: yay! found 3 shares at once
[10:54:16] rejected(0/2)        diff 3.34M      GPU#5   (67 ms)
[10:54:16] reason: bad share
[10:54:16] GPU #2: yay! found 5 shares at once
[10:54:16] rejected(0/3)        diff 759k       GPU#5   (169 ms)
[10:54:16] reason: bad share
[10:54:16] rejected(0/4)        diff 596k       GPU#5   (170 ms)
[10:54:16] reason: bad share
[10:54:16] oh my gosh! Too many rejects, disconnect
[10:54:16] rejected(0/5)        diff 494k       GPU#2   (159 ms)
[10:54:16] reason: bad share
[10:54:16] rejected(0/6)        diff 952k       GPU#2   (159 ms)
[10:54:16] reason: bad share
[10:54:16] rejected(0/7)        diff 565k       GPU#2   (160 ms)
[10:54:16] reason: bad share
[10:54:16] rejected(0/8)        diff 1.73M      GPU#2   (160 ms)
[10:54:16] reason: bad share
[10:54:17] GPU #7: yay! found 6 shares at once
[10:54:17] GPU #5: yay! found 4 shares at once
[10:54:17] GPU #2: yay! found 6 shares at once
[10:54:18] GPU #6: precompute finished
[10:54:18] GPU #4: precompute finished
[10:54:18] GPU #3: precompute finished
[10:54:22] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:54:22] Stratum set raw difficulty to 0.0001
[10:54:22] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.20G
[10:54:22] block: 341,695       job target: 0x0000270fd8f00000
[10:54:23] GPU #6: yay! found 6 shares at once
[10:54:23] oh my gosh! Too many rejects, disconnect
[10:54:23] GPU #7: yay! found 4 shares at once
[10:54:23] GPU #2: yay! found 4 shares at once
[10:54:24] GPU #6: yay! found 7 shares at once
[10:54:24] GPU #4: yay! found 6 shares at once
[10:54:24] GPU #3: yay! found 6 shares at once
[10:54:29] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:54:29] Stratum set raw difficulty to 0.0001
[10:54:29] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.20G
[10:54:29] block: 341,695       job target: 0x0000270fd8f00000
[10:54:30] GPU #6: yay! found 5 shares at once
[10:54:30] rejected(0/9)        diff 2.50M      GPU#6   (61 ms)
[10:54:30] reason: bad share
[10:54:30] rejected(0/10)       diff 503k       GPU#6   (158 ms)
[10:54:30] reason: bad share
[10:54:30] rejected(0/11)       diff 1.51M      GPU#6   (159 ms)
[10:54:30] reason: bad share
[10:54:30] rejected(0/12)       diff 784k       GPU#6   (160 ms)
[10:54:30] reason: bad share
[10:54:30] oh my gosh! Too many rejects, disconnect
[10:54:30] GPU #7: yay! found 5 shares at once
[10:54:30] GPU #5: yay! found 5 shares at once
[10:54:30] GPU #2: yay! found 2 shares at once
[10:54:30] GPU #6: yay! found 5 shares at once
[10:54:31] GPU #3: yay! found 4 shares at once
[10:54:31] GPU #4: yay! found 6 shares at once
[10:54:36] use pool stratum-eu.rplant.xyz:7030 146.59.217.34
[10:54:36] Stratum set raw difficulty to 0.0001
[10:54:36] new job from stratum-eu.rplant.xyz:7030 diff 430k/30.20G
[10:54:36] block: 341,695       job target: 0x0000270fd8f00000
[10:54:37] GPU #6: yay! found 4 shares at once
[10:54:37] rejected(0/13)       diff 440k       GPU#6   (71 ms)
[10:54:37] reason: bad share
[10:54:37] GPU #5: yay! found 3 shares at once
[10:54:37] rejected(0/14)       diff 470k       GPU#6   (176 ms)
[10:54:37] reason: bad share
[10:54:37] rejected(0/15)       diff 467k       GPU#6   (178 ms)
[10:54:37] reason: bad share
[10:54:37] rejected(0/16)       diff 479k       GPU#6   (180 ms)
[10:54:37] reason: bad share
[10:54:37] GPU #7: yay! found 6 shares at once
[10:54:37] oh my gosh! Too many rejects, disconnect
[10:54:37] rejected(0/17)       diff 1.32M      GPU#5   (119 ms)
[10:54:37] reason: bad share
[10:54:37] rejected(0/18)       diff 1.19M      GPU#5   (120 ms)
[10:54:37] reason: bad share
[10:54:37] GPU #2: yay! found 7 shares at once
[10:54:38] GPU #6: yay! found 3 shares at once
[10:54:38] GPU #3: yay! found 3 shares at once
[10:54:38] GPU #4: yay! found 6 shares at once
[10:54:38] GPU #5: yay! found 3 shares at once
[10:54:38] GPU #7: yay! found 6 shares at once

**Comment by andru-kun** [comment]
what if exclude 3080, gpu number 6?

**Comment by andru-kun** [comment]
and man, use tag code, those logs hard to read )

**Comment by deocP** [comment]
OK, so without 3080 not running- I run on 3060Ti  -d 2,5,7 and goes well.
I attach log, for running without 3080 and at the end with 3060Ti only. it's run, so I try to --multi instances and run 3080 separatly, and third instance 2060 wit 1660s 


[deoc — kopia.txt](https://github.com/andru-kun/wildrig-multi/files/9038071/deoc.kopia.txt)

**Comment by deocP** [comment]
[](url)
[1660s20603080.txt](https://github.com/andru-kun/wildrig-multi/files/9038180/1660s20603080.txt)
[3060 — kopia.txt](https://github.com/andru-kun/wildrig-multi/files/9038186/3060.kopia.txt)

**Comment by andru-kun** [comment]
fixed in 0.31.9
